README.rst

```diff
@@ -55,7 +55,15 @@ Get a `Zyte API`_ key, and add it to your project settings.py:
 Instead of adding API key to setting.py you can also set
 ``ZYTE_API_KEY`` environment variable.
 
-Then, set up the scrapy-zyte-api integration:
+Then, set up the scrapy-zyte-api integration. If your Scrapy version supports addons:
+
+.. code-block:: python
+
+    ADDONS = {
+        "scrapy_zyte_api.Addon": 1,
+    }
```

Comment:
There is a note about transparent mode later; I think we should make sure it's written in a way which makes it clear it doesn't apply for addon config.

Reply:
Done, though I'm not sure about the wording.